plugins/anx-udev/README.md

```diff
@@ -0,0 +1,38 @@
+Analogix Device Support
+==============
+
+Introduction
+------------
+
+This plugin can flash the firmware of Analogix device.
+
+Firmware Format
+---------------
+
+The daemon will decompress the cabinet archive and extract a firmware blob in
+an unspecified binary file format. The binary file has a vendor-specific header
+that is used when flashing the image.

```

Comment:
I think we need a lot more detail here. It looks a lot like Intel HEX format. We need enough data to be able to recreate a Analogix firmware file for any content so we can fuzz the format. If it is just intel hex with some kind of extra header then we should probably use a subclass of FuIhexFirmware although without more details I don't know.